--- a/tools/Linux/FEH.py
+++ b/tools/Linux/FEH.py
@@ -168,9 +168,3 @@
 
     if "--no-test" in sys.argv:
         exit(0)
-
-    if (badDirectRendering()):
-        error("XBMC needs hardware accelerated OpenGL rendering.\nInstall an appropriate graphics driver.\n\nPlease consult XBMC Wiki for supported hardware\nhttp://wiki.xbmc.org/?title=Supported_hardware")
-
-    if (badColorDepth()):
-        error("XBMC cannot run unless the\nscreen color depth is atleast 24 bit.\n\nPlease reconfigure your screen.")
